pkgs/top-level/python-packages.nix

```diff
@@ -773,13 +773,18 @@ in modules // {
   };
 
   application = buildPythonPackage rec {
-    name = "python-application-${version}";
-    version = "1.5.0";
+    pname = "python-application";
+    name = "${pname}-${version}";
+    version = "2.0.2";
+    disabled = isPy3k;
```

Comment:
Py3k is not supported, and although the version currently in nixpkgs is built for it this does not work. 
Running it results in "SyntaxError: Missing parentheses in call to 'print')"